assets/dev/js/editor/elements/views/behaviors/sortable.js

```diff
@@ -51,7 +51,7 @@ SortableBehavior = Marionette.Behavior.extend( {
 
 		var $childViewContainer = this.getChildViewContainer(),
 			defaultSortableOptions = {
-				connectWith: $childViewContainer.selector,
+				connectWith: $childViewContainer,
```

Comment:
Does it still work with other containers?

Reply:
Explanation?

Reply:
The previous code affected other containers and allowed elements to be dragged to different areas (e.g. between sections). The new code seems to break this behavior.